### PR TITLE
Transition to CSR from SSR

### LINKED ISSUE #29: move to CSR from SSR
Remove Apollo, razzle etc. 

Use pure webpack or parcel as bundler.

Introduce TS

### PR DESCRIPTION
**Pull request by halvardssm**
closes #29

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,135 @@
...
+# auto-import.
+# .idea/modules.xml
+# .idea/*.iml
+# .idea/modules
+
+# CMake
+cmake-build-*/
+
+# Mongo Explorer plugin
+.idea/**/mongoSettings.xml
```
Comment (Tschakabaka) [inline .gitignore:44]: You can just gitginore **.idea/** to ignore the whole folder. Or do you want to keep commiting some of the files for some reason?

File: README.md [hunk 2]
```diff
@@ -60,194 +31,17 @@ Runs the compiled app in production.
...
-* [facebookincubator/create-react-app](https://github.com/facebookincubator/create-react-app)
-* [humblespark/sambell](https://github.com/humblespark/sambell)
-* [zeit/next.js](https://github.com/zeit/next.js)
-
-#### Author
-
-* [Jared Palmer](https://twitter.com/jaredpalmer)
+* [Felix Dürrwald]()
+* [Halvard Mørstad]()
+* [Marc]()
```
Comment (Tschakabaka) [inline README.md:43]: you forgot the Shark in my name.

File: src/pages/Home.tsx [hunk 3]
```diff
@@ -1,20 +1,21 @@
 import React from 'react'
 
-import {
-  ArtistCard,
-  Featured,
-  GenericList,
-  Hero,
-  VideoCard,
-} from '../components'
+// import {
```
Comment (Tschakabaka) [inline src/pages/Home.tsx]: Im generally against committing code that is commented out unless there is something like a **TODO:** provided

File: src/pages/Home.tsx [hunk 4]
```diff
@@ -51,26 +52,26 @@ function Home() {
...
-            <VideoCard key={event.id} event={event} />
-          ))}
-        </GenericList>
-      </div>
-      {featured && <Featured />}
-      {categories.length > 0 &&
-        categories.map(category => (
-          <Category key={category.id} category={category} />
-        ))}
+      {/*<Hero />*/}
```
Comment (Tschakabaka) [inline src/pages/Home.tsx]: same as above

File: .gitignore (hunk 1 above)
Comment (halvardssm) [reply .gitignore:44]: I used https://gitignore.io to get the `.gitignore`, but I agree, its definetly possible to strip it down 👍

File: src/pages/Home.tsx (hunk 3 above)
Comment (halvardssm) [reply src/pages/Home.tsx]: This is not commented out, I believe GitHub showed you an outdated version of the file 🤔

**Pull request merged**